test/jdk/ProblemList.txt

```diff
@@ -806,6 +806,7 @@ java/awt/image/VolatileImage/VolatileImageConfigurationTest.java 8171069 macosx-
 java/awt/Modal/InvisibleParentTest/InvisibleParentTest.java 8172245 linux-all
 java/awt/Frame/FrameStateTest/FrameStateTest.java 8203920 macosx-all,linux-all
 java/awt/print/PrinterJob/ScaledText/ScaledText.java 8231226 macosx-all
+java/awt/print/PrinterJob/PrintTextTest.java 8148334 generic-all
```

Comment:
Looks like the test has issues (test window vs print to pdf/printer- glyphs look different) on all platforms and problem listing on all platforms (generic-all) is necessary.

Reply:
Correct, that will be JDK-8148334. I do plan to have a look at that one once this PR is merged, but I can't guarantee I'll be able to find the necessary fix (I'm still a bit new to the code), and if I do find the fix I will need some help with testing on Windows and macOS (I only have access to Linux).

Reply:
> Correct, that will be JDK-8148334. I do plan to have a look at that one once this PR is merged,

Right, your changes look good for this PR. The above comment was added as future reference (note to self) since @aivanov-jdk and I observed glyph differences when we tested on different platforms.